src/main/java/org/jboss/pnc/deliverablesanalyzer/CacheProvider.java

```diff
@@ -94,16 +92,12 @@ public DefaultCacheManager initCaches() throws IOException {
             throw new IOException("Cache location is not writable: " + locationPath);
         }
 
-        KojiBuild.KojiBuildExternalizer kojiBuildExternalizer = new KojiBuild.KojiBuildExternalizer();
-        LocalFile.LocalFileExternalizer localFileExternalizer = new LocalFile.LocalFileExternalizer();
         GlobalConfigurationChildBuilder globalConfig = new GlobalConfigurationBuilder();
 
         globalConfig.globalState()
                 .persistentLocation(location)
                 .serialization()
-                .marshaller(new GenericJBossMarshaller())
-                .addAdvancedExternalizer(kojiBuildExternalizer.getId(), kojiBuildExternalizer)
-                .addAdvancedExternalizer(localFileExternalizer.getId(), localFileExternalizer)
+                .addContextInitializer(new ProtobufSerializerImpl())
```

Comment:
This is causing a failure and I don't think you meant it to be part of this PR anyway.

Reply:
It's causing a failure because the BuildFinder snapshot was not yet pushed into sonatype

Reply:
OK, but it's not format related. I would split it into a separate PR even though it's one line.

Reply:
Agree